contracts/pools/PoolCollection.sol

```diff
@@ -884,13 +925,14 @@ contract PoolCollection is IPoolCollection, Owned, BlockNumber, Utils {
                 bntProtocolHoldingsDelta: output.q,
                 baseTokensWithdrawalFee: output.v,
                 baseTokensWithdrawalAmount: baseTokensWithdrawalAmount,
+                poolTokenAmount: poolTokenAmount,
```

Comment:
I was trying to avoid involving the pool tokens in withdrawal amount calculations - why do we need it here?

Reply:
OCD, mostly, since we are already passing almost everything else using this struct (due to `stack too deep` issues) and this amount needs to be passed down anyway

Reply:
ok, makes sense, as long as we don't start adding it to `calculateWithdrawalAmounts`